apis/llms_resource.py

```
@@ -14,29 +15,30 @@ def data_generator(response):
         data = request.json
-        if data.get('stream') == "True":
-            data['stream'] = True # convert to boolean
-        # if not data:
-
-        #     return jsonify({"error": "Invalid request data"}), 400
         try:
             if data.get('stream') == "True":
                 data['stream'] = True # convert to boolean
             # pass in data to completion function, unpack data
             response = completion(**data)
             if data['stream'] == True: 
```

Ruff (0.13.3)

Avoid equality comparisons to `True`; use `data['stream']:` for truth checks

Replace with `data['stream']`

(E712)